src/corporacreator/utils.py

```diff
@@ -0,0 +1,51 @@
+import re
+from typing import Pattern
+
+from num2words import num2words
+
+
+NUMS_REGEX = re.compile(r"(\d+,?\u00A0?\d+)|(\d+\w+)|(\d)+")
+FIND_MULTIPLE_SPACES_REG = re.compile(r'\s{2,}')
+FIND_PUNCTUATIONS_REG = re.compile(r"[/°\-,;!?.()\[\]*…—«»]")
+
+
+def get_numbers(text):
+    return NUMS_REGEX.split(text)
+
+
+def replace_numbers(inp: str, locale: str, ordinal_regex: Pattern = None):
+    finalinp = ''
+    for e in get_numbers(inp):
+        if not e:
+            continue
+        newinp = e
+        try:
+            ee = ''.join(e.split())
+            if int(e) >= 0:
+                newinp = num2words(int(ee), lang=locale)
+        except ValueError:
+            try:
+                ee = ''.join(e.replace(',', '.').split())
+                if float(ee):
+                    newinp = num2words(float(ee), lang=locale)
```

Comment:
Similarly here. How can this work in all cases?

There are several valid readings of "759.11". For example "seven fifty nine point eleven", "seven fifty nine point one one", "seven hundred fifty nine point eleven", "seven hundred fifty nine and eleven hundredths"....


`client_id` is passed to `fr()` so you can hear what the person said and provide the correct transcript and not have to guess.

Reply:
I think we should do that on a `client_id` basis, as you said, but that we should also take the opportunity to fix the current commonvoice dataset to avoid keeping the same bogus numbers over and over.

Reply:
I agree. 
since we can't predict for sure what has was said we should discard such examples.
maybe it can be done in another PR (not handle by existing code anyway)